lib/grplatt.gi

```diff
@@ -923,7 +923,8 @@ InstallGlobalFunction(LatticeViaRadical,function(arg)
       HN:=Image(hom,H);
       c:=LatticeByCyclicExtension(f,
 	  [u->IsSubset(HN,u),u->IsSubset(HN,u)])!.conjugacyClassesSubgroups;
-    elif select<>fail and (select=IsPerfectGroup  or select=IsSimpleGroup) then
+    elif select<>fail and (select=IsPerfectGroup  or select=IsNonabelianSimpleGroup)
```

Comment:
Can't this be simplified? (I am not actually requesting that you do that, though, just wondering; it is fine to merge this PR as-is):
```suggestion
    elif select=IsPerfectGroup or select=IsNonabelianSimpleGroup then
```

Reply:
Yes, it could be simplified. (I think it is a relic from an older construct). I'll do so.